src/components/TaskView.tsx

```diff
@@ -33,10 +33,10 @@ export const TaskView = ({ task, locales }: JobProps) => {
   }
 
   return (
-    <Stack space={3}>
-      <Text weight="semibold" size={1}>
-        Current job progress
-      </Text>
+    <Stack space={4}>
+      <Heading as="h2" weight="semibold" size={2}>
```

Comment:
```suggestion
      <Text as="h2" weight="semibold" size={2}>
```
See above comments on standards in Heading vs Text